README.md

```diff
@@ -168,7 +168,7 @@ Major Contributors
 
 - Andrew Werner [HoeenHero] - Development
 - Austin Couturier [Austin] - Development (damage calculator)
-- [jumbowhales] - Development
+- [peach] - Development
```

Comment:
This is in the wrong place now :)

Reply:
Ah. There were a couple other places not alphabetized either, fixed.